java/clients/src/main/java/sleeper/clients/teardown/ShutdownSystemProcesses.java

```diff
@@ -62,6 +67,12 @@ private void stopECSTasks(InstanceProperties instanceProperties, List<InstancePr
         extraClusters.forEach(clusterName -> stopTasks(ecs, instanceProperties, clusterName));
     }
 
+    private void stopEMRClusters(InstanceProperties properties) throws InterruptedException {
+        if (properties.getList(OPTIONAL_STACKS).contains("EmrBulkImportStack")) {
+            new TerminateEMRClusters(emrClient, properties).run();
+        }
```

Comment:
This check in the if statement might be more trouble than it's worth. If we changed the way bulk import is deployed, this might be difficult to find.